### PR TITLE
Update README.md

### LINKED ISSUE #40: Add repo size badge
```
[![Github Repo Size](https://img.shields.io/github/repo-size/arshadkazmi42/ak-phrase.js.svg)](https://github.com/arshadkazmi42/ak-phrase.js)
```

Add the above line in readme with other badges

### PR DESCRIPTION
**Pull request by marielenna**
Added line 11 next to other badges in the readme file.

Fixes #40

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ It follows the word orders in sentence, same as the words order in array column
 
 <a href="https://www.buymeacoffee.com/arshadkazmi42" target="_blank"><img src="https://www.buymeacoffee.com/assets/img/custom_images/orange_img.png" alt="Buy Me A Coffee" style="height: auto !important;width: auto !important;" ></a>
 <a href="https://www.patreon.com/bePatron?u=15454240" target="_blank"><img src="https://c5.patreon.com/external/logo/become_a_patron_button.png" alt="Become a Patron!" height="40"></a>
+[![Github Repo Size](https://img.shields.io/github/repo-size/arshadkazmi42/ak-phrase.js.svg)](https://github.com/arshadkazmi42/ak-phrase.js)
```
Comment (arshadkazmi42) [inline README.md]: This should be added at line 3, next to downloads badge

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Good start.
There is a small change needed.

File: README.md (hunk 1 above)
Comment (marielenna) [reply README.md]: Thank you @arshadkazmi42 ! I went ahead and removed it and added it to line 3.

```suggestion
[![Github Repo Size](https://img.shields.io/github/repo-size/arshadkazmi42/ak-phrase.js.svg)](https://github.com/arshadkazmi42/ak-phrase.js)
```

File: README.md [hunk 2]
```diff
@@ -9,6 +10,7 @@ It follows the word orders in sentence, same as the words order in array column
 <a href="https://www.buymeacoffee.com/arshadkazmi42" target="_blank"><img src="https://www.buymeacoffee.com/assets/img/custom_images/orange_img.png" alt="Buy Me A Coffee" style="height: auto !important;width: auto !important;" ></a>
 <a href="https://www.patreon.com/bePatron?u=15454240" target="_blank"><img src="https://c5.patreon.com/external/logo/become_a_patron_button.png" alt="Become a Patron!" height="40"></a>
 
+
```
Comment (arshadkazmi42) [inline README.md]: There is a extra line break here.
Can you remove it pls

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Awesome.
Just an extra line break is added, once that is removed it will be ready to get merged

File: README.md (hunk 2 above)
Comment (marielenna) [reply README.md]: Line break removed.
```suggestion

```

**Review by arshadkazmi42 — APPROVED** [review]
:tada:

**Pull request merged**